ecs/README.md

```diff
@@ -1,8 +1,23 @@
-## Alert Logic al-agent-container Deployment for ECS
+## Deploy the Alert Logic Agent Container for ECS
 
-#### Deploy Alert Logic Agent Task Definition
+## Before You Begin
+To deploy the Alert Logic Agent Container for ECS, you will need your unique registration key. 
 
-1. Update the al-agent-ecs.json with your unique registration key.
+**To find your unique registration key:**
+
+1. In the Alert Logic console, click the Support Information icon.
+2. Click "Details."
+3. Copy your unique registration key.
+
+In addition, be sure you install the AWS command line interface (CLI) and  ensure point it to, and configure it for, the appropriate AWS account. 
```

Comment:
If you wish to link some of this text to the AWS CLI install guide, which I think would be nice, here's the official one:

https://aws.amazon.com/cli/